Delegation documentation (#448)
Concordium/concordium.github.io: source/shared/net/desktop-wallet/remove-delegation.rst

If you remove the delegation, there is a cool-down period before the delegation is stopped. During this period, you continue to earn rewards; you cannot change the delegation amount, but you can change other delegation settings. After the cool-down period, the amount you had delegated is returned to your disposable balance.

.. Note::

   All transfers and transactions cost a fee, including staking and unstaking transactions. The fee is based on the set NRG for that transaction and the current exchange rate.
   The cost of transaction fees is stable in Euros, and therefore the price in CCD varies depending on the CCD to EUR exchange rate. The fee will always be deducted from the **Balance** of the account, so it is important to have some available CCDs to cover fees. A locked-for-staking balance cannot be used to pay for these transactions.
   You can see the fee in the transaction log.

.. Warning::
   Make sure you have enough funds in your disposable balance to cover transaction fees.

Remove delegation (Single-signature account)
============================================

#. Go to **Accounts** and select the account on which you have delegated funds.

> #. Click **Change view** then select *Remove Delegation**.

Reviewer: ```suggestion
#. Click **More options** then select **Delegation** and **Remove delegation**.
```

Remove delegation (Single-signature account)
============================================

#. Go to **Accounts** and select the account on which you have delegated funds.

#. Click **More options** then select **Delegation** and choose **Remove Delegation**.

#. Review the cool-down period. Click **Continue**.

#. A message says **Waiting for device. Please connect your Ledger**. Connect the Ledger device to the computer and enter your PIN on the Ledger device.

#. Press the right button to navigate to the **Concordium** app, and then press both buttons to open the app. The Ledger says **Concordium is ready**. Wait for the message **Ledger Nano S is ready** in the Desktop Wallet and select **Submit**.

#. On the Ledger device, a message says **Review transaction**. The Ledger device shows **Stop delegation**. Navigate to the right.

#. The Ledger device says **Sign transaction**. Press both buttons to sign the transaction. The Ledger device says **Concordium is ready**.

#. In the Desktop Wallet, you can see that the transaction has been submitted to the chain. Select **Finish**.

Remove delegation (Multi-signature account)
===========================================

> #. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Update baker stake**.

Reviewer 2: ```suggestion
#. Go to **Multi Signature Transactions**, select **Make new proposal**, and then select **Remove Delegation**.
```

#. Select the account whose baker stake you want to change.

> #. Enter the new amount that you want to stake and select **Continue**.

Reviewer 2: ```suggestion
#. Review the cool-down period. Click **Continue**.
```
